packages/esm-service-queues-app/src/active-visits/active-visits-table.resource.ts

```diff
@@ -34,21 +30,18 @@ export interface MappedVisitQueueEntry {
   encounters: Array<MappedEncounter>;
   name: string;
   patientAge: string;
```

Comment:
patientAge would always be a **number**. I think patientAge type should be number instead of string.

Reply:
1. You'll note that this property isn't added by this PR.
2. Did you look at the context in which this is used? If you had you might've realised that this will always be a number.

Please read the code you are reviewing. If you are unsure of something, it's better to phrase it as a question, i.e., "Shouldn't this be a number?" or "Why isn't this a number?"